### PR TITLE
Added minor improvements for releases

### PR DESCRIPTION
**Pull request by anfimovdm**
I have some doubts about `func.current_timestamp()`, I think it depends on timezone in DB host, **we need to check it**

### CODE REVIEW

**Review by soksanichenko — APPROVED** [review]

File: alws/crud/release.py [hunk 1]
```diff
@@ -19,24 +19,57 @@
...
+        ).order_by(models.Release.id.desc())
+        if count:
+            query = select(func.count(models.Release.id))
+        if release_id:
+            query = query.where(models.Release.id == release_id)
+        if search_params:
+            # These links could be helpful for filtering by JSON
+            # https://github.com/sqlalchemy/sqlalchemy/discussions/7991
+            # https://www.postgresql.org/docs/9.5/functions-json.html
+            # if search_params.package_name:
```
Comment (soksanichenko) [inline alws/crud/release.py:42]: Should it be commented?

**Review by soksanichenko — COMMENTED** [review]
дпеь

**Comment by javihernandez** [comment]
> I have some doubts about `func.current_timestamp()`, I think it depends on timezone in DB host, **we need to check it**

Right, sqlalchemy's `current_timestamp` calls Postgresql's [CURRENT_TIMESTAMP](https://www.postgresqltutorial.com/postgresql-date-functions/postgresql-current_timestamp/), which includes the timezone offset.

File: alws/crud/release.py (hunk 1 above)
Comment (anfimovdm) [reply alws/crud/release.py:42]: I comment it by purpose, I don't have enough time right now for researching how to make a query for JSON field, so I provide some links for the future

File: alws/alembic/versions/76855243a038_added_timestamp_for_releases.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+revision = '76855243a038'
+down_revision = 'ec505e94fd1a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('build_releases', sa.Column('created_at', sa.DateTime(), nullable=True))
+    op.alter_column('build_releases', 'product_id',
```
[the remark this answers is not in the document]
Comment (anfimovdm) [reply alws/alembic/versions/76855243a038_added_timestamp_for_releases.py:22]: I'm not sure, but it appears after population `Product` entities for releases, at first it was nullable, but after it became not nullable

**Review by javihernandez — APPROVED** [review]

**Review by maccelf — APPROVED** [review]

**Review by soksanichenko — APPROVED** [review]

**Review by javihernandez — APPROVED** [review]

**Pull request merged**